pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+<h4> Issue(Describe in 2-3 lines what the current issue or background is) : </h4>
```

Comment:
Following the Google CL Description style guide recommendation - https://engdoc.corp.google.com/eng/doc/devguide/coding/review/developer/cl-descriptions.md?cl=head, first line is usually the short summary about what is being done - go/cl-descriptions#firstline. For our case, we can have "Summary" tag here instead